test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxserviceTests.cs

```diff
@@ -266,20 +268,21 @@ private async Task<Mock<RequestContext<DacFxResult>>> SendAndValidateGenerateDep
             var generateScriptParams = new GenerateDeployScriptParams
             {
                 PackageFilePath = extractParams.PackageFilePath,
-                DatabaseName = string.Concat(sourceDb.DatabaseName, "-deployed"),
-                ScriptFilePath = Path.Combine(folderPath, string.Concat(sourceDb.DatabaseName, "_", "UpgradeDACScript.sql"))
+                DatabaseName = targetDb.DatabaseName
             };
 
+            // Generate script for deploying source dacpac to target db
             GenerateDeployScriptOperation generateScriptOperation = new GenerateDeployScriptOperation(generateScriptParams, result.ConnectionInfo);
             service.PerformOperation(generateScriptOperation);
-            SqlTestDb targetDb = SqlTestDb.CreateFromExisting(generateScriptParams.DatabaseName);
+
+            // Verify script was generated
+            Assert.NotEmpty(generateScriptOperation.Result.DatabaseScript);
+            Assert.Contains("CREATE TABLE", generateScriptOperation.Result.DatabaseScript);
 
             // cleanup
-            VerifyAndCleanup(generateScriptParams.ScriptFilePath);
             VerifyAndCleanup(extractParams.PackageFilePath);
             sourceDb.Cleanup();
             targetDb.Cleanup();
```

Comment:
Not specifically for this change but please add source db and target db cleanup in a finally block so that its cleaned up even if something goes wrong...

Reply:
added try{} finally{} to all the dacfx tests